sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/AnomalyDetection/AnomalyIncident.cs

```diff
@@ -77,5 +79,19 @@ internal AnomalyIncident(string metricId, string detectionConfigurationId, strin
         /// </summary>
         [CodeGenMember("IncidentStatus")]
         public AnomalyIncidentStatus Status { get; }
+
+        /// <summary>
+        /// The value of the data point at the root node of this incident. The root node is defined as
+        /// the data point at the root of this incident's root-cause analysis tree.
+        /// </summary>
+        public double ValueOfRootNode { get; }
+
+        /// <summary>
+        /// The value the data point at the root node of this incident was expected to have according to
+        /// the service's smart detector. The root node is defined as the data point at the root of this
+        /// incident's root-cause analysis tree. <c>null</c> if the quantity of historical points is not
+        /// enough to make a prediction.
+        /// </summary>
+        public double? ExpectedValueOfRootNode { get; }
```

Comment:
The new properties are hard to test because we can't really validate what the service is sending back to us. We could get `default(double)` and `default(double?)` with no problem, and this would still be a valid service response. Maybe consider mocking?